docs/user-guide/migrating-from-jscs.md

```diff
@@ -0,0 +1,79 @@
+# Migrating from JSCS
+
+In April 2016, we [announced](http://eslint.org/blog/2016/04/welcoming-jscs-to-eslint) that the JSCS project was shutting down and the JSCS team would be joining the ESLint team. This guide is intended to help those who are using JSCS to migrate their settings and projects to use ESLint. We've tried to automate as much of the conversion as possible, but there are some manual changes that are needed.
+
+## Terminology
```

Comment:
Here is a small prototoype using jscodeshift (recast):
https://astexplorer.net/#/LjSW9EYcUP/2
Obviously it's doing it in a stupid way but just for the sake of an example.
Also, I agree that this should be a separate tool or part of polyjuice.